packages/frontend/src/views/HomeView.vue

```diff
@@ -1,7 +1,40 @@
 <template>
-  <div>This is a cool home page maybe</div>
+  <div class="grid pt-6">
+    <div class="col-6">
+      <h1 class="text-6xl line-height-2 font-semibold mb-2">Help improve Open-Source sustainability!</h1>
+      <h4 class="text-2xl font-light mb-4">
+        A lot of Open-Source projects suffer from financial sustainability and face the risk of not getting maintained
+        due to a lack of resources. Use the Dependency Health Monitor to view the financial state of your dependencies
+        and support your dependency!
+      </h4>
+      <Button @click="redirectToExecuteReportPage" label="Execute the report!" icon="pi pi-heart" />
+    </div>
+    <div id="help-improve" class="col-6">
+      <IconHelpImproveVue />
+      <div class="flex justify-content-center">
+        <Button
```

Comment:
Can you replace this with an `<a href="https://storyset.com/people" target="_blank" rel="noopener noreferer">`?

Reply:
Resolved in a10417f5bc57f84d48c6ff15e83be98ba7ad8e6f